xml/obs_notifications.xml

```diff
@@ -0,0 +1,349 @@
+     <emphasis>Requests:</emphasis>
+     All unread notifications related to requests will be displayed when
+     selecting this filter.
+    </para>
+   </listitem>
+
+   <listitem>
+    <para>
+     <emphasis>Incoming Requests:</emphasis>
+     All unread notifications related to incoming requests, so requests which
+     someone submitted to a project/package you are involved in, will be
+     displayed when selecting this filter.
+    </para>
+   </listitem>
+
+   <listitem>
+    <para>
+     <emphasis>Outgoing Requests:</emphasis>
+     All unread notifications related to outgoing requests, so requests which
+     you submitted to a project/package, will be displayed when selecting this
+     filter.
+    </para>
+   </listitem>
+
+   <listitem>
+    <para>
+     <emphasis>Projects:</emphasis>
+     All unread notifications related to a project will be displayed when
+     selecting a project filter. Each project with at least one unread
+     notification has its own project filter.
+    </para>
+   </listitem>
+
+   <listitem>
+    <para>
+     <emphasis>Groups:</emphasis>
+     All unread notifications related to a group will be displayed when
+     selecting a group filter. Each group with at least one unread notification
+     has its own group filter.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="cha.obs.notifications.api">
+   <title>API</title>
+   <para>
+     Every user can check their unread notifications by querying:
+   </para>
+   <screen>osc api -X GET "/my/notifications"</screen>
+   <para>
+     It will return a list with your unread notifications:
+   </para>
+   <screen language="xml">
+&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+&lt;notifications count="2"&gt;
+    &lt;total_pages&gt;1&lt;/total_pages&gt;
+    &lt;current_page&gt;0&lt;/current_page&gt;
+    &lt;notification id="3"&gt;
+        &lt;title&gt;test 1&lt;/title&gt;
+        &lt;who&gt;Administrator&lt;/who&gt;
+        &lt;state&gt;new&lt;/state&gt;
+        &lt;when/&gt;
+        &lt;event_type&gt;review_wanted&lt;/event_type&gt;
+    &lt;/notification&gt;
+    &lt;notification id="25"&gt;
+        &lt;title&gt;test 2&lt;/title&gt;
+        &lt;who&gt;User 2&lt;/who&gt;
+        &lt;event_type&gt;comment_for_package&lt;/event_type&gt;
+        &lt;when/&gt;
+    &lt;/notification&gt;
+&lt;/notifications&gt;
+   </screen>
+   <para>
+     This one will toggle a notification read or unread.
+   </para>
+   <screen>osc api -X PUT "/my/notifications/3"</screen>
+   <para>
+     It will say if the operation went well or not.
+   </para>
+   <screen language="xml">
+&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+&lt;status code="ok"&gt;
+    &lt;summary&gt;Ok&lt;/summary&gt;
+&lt;/status&gt;
+   </screen>
+   <para>
+     To check required parameters and options, please check our throughly documented <link xlink:href="https://build.opensuse.org/apidocs-new/#/Notifications/get_my_notifications">Experimental API Documentation</link>.
```

Comment:
```suggestion
     To check required parameters and options, please check our thoroughly documented <link xlink:href="https://build.opensuse.org/apidocs-new/#/Notifications/get_my_notifications">Experimental API Documentation</link>.
```

Reply:
@danidoni I wouldn't call it experimental, I think its already a bit more then that :) also you twice use the word `check`

Maybe something like: `For more information regarding the notifications API endpoints, checkout the API Documentation

Reply:
or maybe `checkout the developer documentation`

Reply:
> @danidoni I wouldn't call it experimental, I think its already a bit more then that :)

Sure, but the "official API documentation" is still another one. Mentioning only "API documentation" doesn't include the idea of a not yet officially approved documentation. I would keep the sentence with "Experimental API Documentation".

Reply:
okay, makes sense, would still rephrase it a bit, just to make the sentence a bit nicer, maybe `For more information regarding the notifications API endpoints, checkout the Experimental API Documentation` ?